### PR TITLE
Jetpack Cloud: Add Jetpack VideoPress Product

### PR DESCRIPTION
**Pull request by roundhill**
#### Changes proposed in this Pull Request

* Adds VideoPress as a new product to the pricing grid
* Moves CRM card to the bottom area alongside `Jetpack Free`
* Adds a thank you dialog to be displayed after purchase of Jetpack VideoPress product.

Screenshots:
jetpack cloud localhost_3000_pricing (1)

Screen Shot 2021-09-15 at 12 22 10 PM

Screen Shot 2021-09-15 at 12 07 29 PM

#### Testing instructions

* `CALYPSO_ENV=jetpack-cloud-development yarn start`
* View `/pricing`, and you should see VideoPress as an option.
* Clicking the `Get VideoPress` link should take you to the VideoPress checkout.
* After completing the purchase, kill the server and run `yarn start` to start regular calypso.
* Simulate the `Thank You` experience by loading: `http://calypso.localhost:3000/plans/my-plan/{your-jp-site-slug}?product=jetpack_videopress&thank-you`
* Visit `http://calypso.localhost:3000/plans/my-plan/{your-jp-site-slug}` and you should see Jetpack VideoPress in the `My solutions` section.

Related to pxWta-Zz-p2

### CODE REVIEW

File: client/lib/plans/features-list.js [hunk 1]
```diff
@@ -1317,13 +1322,38 @@ export const FEATURES_LIST = {
 		getSlug: () => FEATURE_LIVE_CHAT_SUPPORT_BUSINESS_DAYS,
 		getTitle: () => i18n.translate( 'Live chat support 24X5' ),
 		getDescription: () =>
-			i18n.translate( 'Live chat is available 24 hours a day from Monday through Friday.' ),
+			i18n.translate( 'Live chat is available 24 hours a day from Monday through Friday' ),
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/lib/plans/features-list.js [hunk 2]
```diff
@@ -1317,13 +1322,38 @@ export const FEATURES_LIST = {
...
 	},
 
 	[ FEATURE_LIVE_CHAT_SUPPORT_ALL_DAYS ]: {
 		getSlug: () => FEATURE_EMAIL_LIVE_CHAT_SUPPORT_ALL_DAYS,
 		getTitle: () => i18n.translate( 'Live chat support 24X7' ),
-		getDescription: () => i18n.translate( 'Live chat is available 24/7.' ),
+		getDescription: () => i18n.translate( 'Live chat is available 24/7' ),
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/plans/jetpack-plans/slug-to-selector-product.ts [hunk 3]
```diff
@@ -122,6 +129,14 @@ function itemToSelectorProduct(
 						)
 					) || [],
 			},
+			// We need to hack VideoPress a bit as it has a free option.
+			...( isVideoPress && {
+				isFree: true,
+				externalUrl: 'https://jetpack.com/features/design/video-hosting',
+				belowPriceText: translate( 'from %(minPrice)s - %(maxPrice)s/month', {
```
Comment (a8ci18n) [inline client/my-sites/plans/jetpack-plans/slug-to-selector-product.ts]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'from %(minPrice)s - %(maxPrice)s' )` *ES Score: 8*

File: packages/calypso-products/src/translations.js [hunk 4]
```diff
@@ -140,6 +147,7 @@ export const getJetpackProductsCallToAction = () => {
 	const backupT2 = translate( 'Get Backup Pro' );
 	const search = translate( 'Get Site Search' );
 	const scan = translate( 'Get Scan' );
+	const videoPress = translate( 'Get VideoPress' );
```
Comment (a8ci18n) [inline packages/calypso-products/src/translations.js:154]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Get started with VideoPress' )` *ES Score: 7*

File: packages/calypso-products/src/translations.js [hunk 5]
```diff
@@ -168,6 +178,7 @@ export const getJetpackProductsTaglines = () => {
 	const scanTagline = translate( 'Protect your site' );
 	const scanOwnedTagline = translate( 'Your site is actively being scanned for malicious threats' );
 	const antiSpamTagline = translate( 'Block spam automatically' );
+	const videoPressTagLine = translate( 'High-quality, ad-free video for WordPress' );
```
Comment (a8ci18n) [inline packages/calypso-products/src/translations.js:185]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Host fast, high-quality, ad-free video.' )` *ES Score: 7*

File: client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx [hunk 6]
```diff
@@ -75,6 +76,12 @@ const JetpackComMasterbar: React.FC = () => {
 								href: '/boost/',
 								icon: boostIcon,
 							},
+							{
+								label: translate( 'VideoPress' ),
+								description: translate( 'High-quality, ad-free video' ),
```
Comment (a8ci18n) [inline client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx:84]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Host fast, high-quality, ad-free video.' )` *ES Score: 9*

**Comment by pgk** [comment]
Overall it tested as expected for me. I guess we might want to have all checks pass before we merge

**Review by pgk — CHANGES_REQUESTED** [review]
Code-wise it LGTM, let's fix those tests though

**Comment by monsieur-z** [comment]
Hi @roundhill! I'm seeing a different card than what's shown in the screenshot when I run the branch locally. I applied the patches mentioned in [here](https://github.com/Automattic/jetpack/pull/21006) to my sandbox. Did I maybe miss something?
Screen Shot 2021-09-21 at 9 57 03 AM

**Comment by roundhill** [comment]
> Hi @roundhill! I'm seeing a different card than what's shown in the screenshot when I run the branch locally. I applied the patches mentioned in [here](https://github.com/Automattic/jetpack/pull/21006) to my sandbox. Did I maybe miss something?

Oh, yes the original screenshot is outdated. @robertbpugh sent some changes to the card copy over slack, and wanted to show the price instead of `Free`.

**Review by jgcaruso — COMMENTED** [review]
This tested well for me too.

**Comment by roundhill** [comment]
@pgk a rebase got the tests working 👍

File: client/lib/plans/features-list.js [hunk 7]
```diff
@@ -1315,6 +1319,26 @@ export const FEATURES_LIST = {
...
 	},
+
+	[ FEATURE_JETPACK_VIDEOPRESS ]: {
+		getSlug: () => FEATURE_JETPACK_VIDEOPRESS,
+		getTitle: () => i18n.translate( 'Unlimited users' ),
+	},
+
+	[ FEATURE_JETPACK_VIDEOPRESS_EDITOR ]: {
+		getSlug: () => FEATURE_JETPACK_VIDEOPRESS_EDITOR,
+		getTitle: () => i18n.translate( 'Built into WordPress editor' ),
```
Comment (a8ci18n) [inline client/lib/plans/features-list.js:1329]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Built into WordPress Editor' )` *ES Score: 10*

**Review by pgk — APPROVED** [review]
tests passed!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6731177

Thank you @roundhill for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6731177).

**Pull request merged**